### ISSUE TITLE
Set HD wallet as the default wallet type

### ISSUE THREAD

**Issue by forro68** [dashpay/dash#3469]
HD wallets have existed since dash core version 0.12..2.0.
https://docs.dash.org/en/stable/wallets/dashcore/advanced.html#hd-wallets

This should be the default type. This way users only have to save the seed words, and never need to worry about making backups of their wallet.dat files.
This will allow other functionality, such as setting multi-session privatesend enabled by default.

Curious as to why this has not yet happened. Good grief.

**Comment by PastaPastaPasta** [comment]
See https://github.com/dashpay/dash/pull/2419.

A lot of backports are needed to make this possible

**Comment by forro68** [comment]
If I understand correctly, the simple solution is to encrypt the wallet upon creation. After launching the dash core app for the first time, provide a popup where the user is asked to create a password, and that it will be used to encrypt the wallet. Wouldn't that do it?

**Comment by PastaPastaPasta** [comment]
yes, pretty much, however actually getting there somewhat difficult. Wallet creation and general startup are pretty entwined. To get to a place where we are able to do this requires a lot of backporting. This is something that we plan on doing, but we aren't there yet.

**Comment by forro68** [comment]
Depends on [#1379](https://github.com/dashpay/dash/issues/1379).
